.github/labeler.yml

```diff
@@ -14,10 +14,26 @@
   - "doc/**/*"
   - "**/*.rst"
 
-"CI-libmodem-test":
+"CI-iot-samples-test":
```

Comment:
Perhaps all the folders here could end with `/**/*` in case they later decide to add subfolders?